### ISSUE TITLE
Can't expand node once dragged

### ISSUE THREAD

**Issue by zainulabidin302** [angular-ui-tree/angular-ui-tree#830]
https://angular-ui-tree.github.io/angular-ui-tree/#/basic-example

Can't expand the node once dragged.
To reproduce the issue please check the behavior in the above link example. 
1. Try to drag a node.
2. Now then try to expand.
   
   You will not be able to do it. It will go to drag state. This problem is with newer versions > 2 . previous versions works good. 

Thanks

**Comment by rlintern** [comment]
we are experiencing same issue (in #basic-example and in our own integration)

**Comment by rlintern** [comment]
This seems to happen when you don't actually end up moving the item in the tree. That is, you temporarily drag it very slightly but make sure it goes back into the same spot.  A simple left-click on a tree node will trigger the same thing (that is, temporary drag into same spot).  Things seem to work fine if you drag a node to new spot in the tree.  Hope that helps to diagnose.
Also be aware that once a node gets in this state, then any clicks on the node and all its descendants will not work as expected whether it be expand/collapse, add, or remove.

**Comment by rlintern** [comment]
Release 2.19.0 does not have this issue.  Releases 2.20.0+ do.

**Comment by jacobscarter** [comment]
Thank you that is helpful, I will start looking into this.

**Comment by tinybright** [comment]
Release 2.22.1 have this issue.

**Comment by jacobscarter** [comment]
@tinybright release 2.22.1 was not supposed to fix this, it is still an open bug ticket. I am working on it and will let you know when I have an update.

**Comment by DominikPalo** [comment]
Also all elements with the attrribute `data-nodrag` are not clickable after ndoe was dragged.

**Comment by m1llen1um** [comment]
git bisect _says_


```
$ git bisect bad
ceef22463a967ab29aa284c00c3ac86d08cac6a6 is the first bad commit
commit ceef22463a967ab29aa284c00c3ac86d08cac6a6
Author: jacobscarter <jacob@ieksolutions.com>
Date:   Fri Sep 9 12:26:21 2016 -0700

Add performance improving code
```

**Comment by m1llen1um** [comment]
This should be a fix - #846. I hope my PRs to be reviewed in the nearest future.

**Comment by ErosBusTravel** [comment]
A temporal fix is add a delay to ui-tree

"data-drag-delay"

that worked for me.

**Comment by Nanakii** [comment]
Seems like it has been fixed in 2.22.5 (PR #878)

**Comment by Voles** [comment]
@Nanakii you're right. Thanks for reporting.
